### PR TITLE
Update: Provide more clarity about the term ANY time for initial development releases

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Description

The text `Anything MAY change at any time` can lead the idea that no rules are required and that PATCH releases can also introduces breaking changes or new functionalities when I understand that the whole idea here is clarifies that no compatible changes will be addressed in MINOR releases since the project has no MAJOR bumps at all. 

So, could we please try to clarifies this one?

Closes: https://github.com/semver/semver/issues/915

### CODE REVIEW

**Comment by ljharb** [comment]
v0 versions have no patch number.

**Comment by camilamacedo86** [comment]
Hi @ljharb,

> v0 versions have no patch number.

So, are you saying that the superset rule is not applied in the initial development. If so, what is the point of work with `0.Y.Z`. Why would we need to have Z releases fi we can distributed to them new features and bracking changes at the same way that we can do with `0.Y` releases?

PATCH version when you make backwards compatible bug fixes

File: semver.md [hunk 1]
```diff
@@ -68,7 +68,9 @@ Each element MUST increase numerically. For instance: 1.9.0 -> 1.10.0 -> 1.11.0.
 MUST NOT be modified. Any modifications MUST be released as a new version.
 
 1. Major version zero (0.y.z) is for initial development. Anything MAY change
-at any time. The public API SHOULD NOT be considered stable.
+at any time. The public API SHOULD NOT be considered stable. Incompatible API changes and 
+new functionality might be introcuded in MINOR releases. PATCH releases still reserved
```
Comment (camilamacedo86) [inline semver.md:72]:
```suggestion
new functionality might be introcuded in incremental `0.Z` releases when `0.Y` releases still reserved
```

**Comment by ljharb** [comment]
Ideally you'd start the project with 1.0.0 and avoid v0 versions entirely :-)

**Comment by camilamacedo86** [comment]
Hi @ljharb,

Thank you for your replies. Let's try to discuss this one in : https://github.com/semver/semver/issues/915 where we can keep all thoughts centralized.

**Comment by jwdonahue** [comment]
I have a problem with:

> The public API SHOULD NOT be considered stable.

Okay, so the API can change but breaking changes in the implementation aren't allowed? 

And this is contradictory:

> 0.Y.Z releases still reserved to backwards compatible bug fixes.

Which is it? Stable or not stable?

It's a prerelease, therefore all bets are off with regard to the meaning of any of the fields in the triple. 

This phrase from the [9th clause](https://semver.org/#spec-item-9) makes sense to me:

> A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version.

The "and might not satisfy compatibility requirements..." part of that, tells me what I need to know about "unstable".

So why not just drop [clause 4](https://semver.org/#spec-item-4)?  I think that is what was suggested by @stevelabnik in #915.

In any case, messing around with how prereleases are detected and what they mean is a breaking change.

**Pull request closed without merging**